### PR TITLE
Adds initial Kubebuilder scaffolding

### PR DESCRIPTION
**Pull request by danehans**
Scaffolds the project with Kubebuilder using `projectcontour.io` as the domain, `operator` as the group, `Contour` as the kind and `v1alpha1` as the version.

/assign @jpeach @Miciah

Signed-off-by: Daneyon Hansen <daneyonhansen@gmail.com>

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Build the manager binary
+FROM golang:1.13 as builder
```
Comment (jpeach) [inline Dockerfile:2]: Might as well begin with Go 1.15 :)

**Review by jpeach — APPROVED** [review]
This looks good; my only suggestion is that we might as well start off with Go 1.15.

**Review by youngnick — APPROVED** [review]
Yeah `operator.projectcontour.io/Contour` LGTM for the apigroup and name.

**Pull request merged**